controls/cf_serverd.cf

```diff
@@ -45,12 +45,25 @@ body server control
       cfruncommand => "$(sys.cf_agent) -I -D cfruncommand -f \"$(sys.update_policy_path)\"  &
                        $(sys.cf_agent) -I -D cfruncommand";
 
-    !windows::
-      # last single quote in cfruncommand is left open, so that
-      # arguments (like -K and --remote-bundles) are properly appended.
-      cfruncommand => "$(def.cf_runagent_shell) -c \'
-                           $(sys.cf_agent) -I -D cfruncommand -f $(sys.update_policy_path)  ;
-                           $(sys.cf_agent) -I -D cfruncommand";
+    !windows.!cfengine_3_7::
+
+      # in cfengine 3.10.0 and later last single quote in cfruncommand is left
+      # open, so that arguments (like -K and --remote-bundles) are properly
+      # appended.
+
+        cfruncommand => "$(def.cf_runagent_shell) -c \'
+                           $(sys.cf_agent) -I --define cfruncommand -f $(sys.update_policy_path)  ;
+                           $(sys.cf_agent) -I --define cfruncommand";
+
+    !windows.cfengine_3_7::
+
+      # 3.7 doesn't support specifying the remote bundlesequence and does not
+      # know to add a trailing single quote so it must be accounted for here.
```

Comment:
No, that's not the difference. 3.10 does not auto-append a closing trailing quote either. But when EOF is reached, the shell closes all quotation implicitly.